overlays/migrate-to-nonroot/kustomization.yaml

```diff
@@ -7,7 +7,6 @@ patchesStrategicMerge:
   - grafana/grafana.Deployment.yaml
   - indexed-search/indexed-search.StatefulSet.yaml
   - lsif-server/lsif-server.Deployment.yaml
-  - pgsql/pgsql.Deployment.yaml
```

Comment:
This is still safe? We can run the init container regardless if this kustomization being run?

Reply:
that initContainer is what the deleted kustomization did